docs/products/compute/compute-instances/guides/local-disk-encryption/index.md

```diff
@@ -0,0 +1,67 @@
+| [**Migration:**](/docs/products/compute/compute-instances/guides/migrate-to-different-dc/) moves your compute instance to another data center. |During migration, a new disk is created on the destination host. Decrypted bits are copied over from the source to the destination. The new disk is encrypted if the destination host has local disk encryption enabled. |
+| [**Rebuild:**](/docs/products/compute/compute-instances/guides/rescue-and-rebuild/) start over with a fresh Linux distribution or use a backup. | You can enable or disable disk encryption during a Rebuild. During a Rebuild, the previous encryption setting is used unless it's changed.<br> **Note:** The **Encrypt Disk** setting for Linodes attached to a node pool can not be changed.</br>|
+| [**Rescue:**](/docs/products/compute/compute-instances/guides/rescue-and-rebuild/) boot your compute instance into Rescue Mode to perform system recovery tasks and transfer data off the disks when you suspect a corrupt file system. | When a rescue image is deployed, and if encryption is enabled, the data stored on the disk is encrypted again.|
+| [**Resize:**](/docs/products/compute/compute-instances/guides/resize/) changing a compute instances plan to resize your instance. |<li>Data on encrypted disks remain encrypted.</li> <li>Data on unencrypted disks remain unencrypted.</li> |
+
+## Considerations
+
+- Disk encryption is currently not available in all regions. Select another region to use Disk Encryption or enable encryption when it does become available using [Rebuild](/docs/products/compute/compute-instances/guides/rescue-and-rebuild/#rebuilding).
+
+- Distributed Compute Instances are encrypted automatically if this feature is supported in the region. The disk encryption setting can not be changed.
+
+- New LKE clusters are encrypted if disk encryption is supported in the region. This disk encryption setting can not be changed.
+
+- Encryption can increase compute instance CPU overhead and decrease realized throughput.
+  - For performance sensitive workloads on linodes that are not part of a node pool, you can opt-out of disk encryption.
+  - For performance sensitive workloads on linodes that are part of a LKE node pool, you can create additional node pools to spread out the workloads if required.
+
+- If the compute instance is part of a LKE node pool, you cannot change the disk encryption setting. If a node pool is not encrypted and you want an encrypted node pool, delete the node pool and create a new node pool. New node pools are always encrypted.
```

Comment:
This I will call out... we should either call them Linodes or Compute Instances, to maintain consistency.